db/fixtures/customization_templates.yml

```diff
@@ -274,3 +274,10 @@
     />\n</unattend>"
   :type: CustomizationTemplateSysprep
   :system: true
+
+- :name: Basic root pass template
+  :description: This template takes use of rootpassword defined in the UI
+  :script: "#cloud-config\nchpasswd:\n  list: |\n    root:<%= MiqPassword.decrypt(evm[:root_password]) %>\n  expire: False"
+  :type: CustomizationTemplateCloudInit
+  :pxe_image_type_id: 1
```

Comment:
Correct, following the pattern of what is already there you want to set `:system` to true and when it is seeded it will be available in the `Examples` area of the Customization Templates.